models/models_users.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../db/connection.js')
+
+function fetchUsers() {
+    return db.query("SELECT username FROM users WHERE username NOT LIKE 'NULL';")
```

Comment:
There shouldn't be any NULL values in the users table - you would be able to omit the WHeRE clause here 😀